src/main/kotlin/no/nav/tilleggsstonader/sak/statistikk/vedtak/VedtaksstatistikkService.kt

```diff
@@ -54,6 +57,7 @@ class VedtaksstatistikkService(
                 vedtaksperioder = VedtaksperioderDvh.fraDomene(stønadsperioder),
                 utbetalinger = UtbetalingerDvh.fraDomene(andelTilkjentYtelse),
                 kravMottatt = behandling.kravMottatt,
+                årsakerAvslag = ÅrsakAvslagDvh.fraDomene(vedtak?.årsakerAvslag),
```

Comment:
```suggestion
                årsakerAvslag = ÅrsakAvslagDvh.fraDomene(vedtak?.årsakerAvslag?.årsaker),
```
Kan batch-committes sammen med [dette forslaget](https://github.com/navikt/tilleggsstonader-sak/pull/325/files#r1611981074)